src/core/transactionmanager.js

```diff
@@ -539,6 +540,10 @@ TransactionManager.prototype._prepareRequest = function(tx) {
     submitRequest.txJson(tx.tx_json);
   }
 
+  if (!_.isEmpty(tx.signers)) {
+    submitRequest.message.Signers = tx.signers;
```

Comment:
Does this actually work for locally signed transactions, i.e. a txn submitted via `tx_blob` property? I'm guessing not